lib/region/EU/countryCodes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file List of country codes legally considered part of the EU region.
+ */
+
+var countryCodes = {
+  AT: true, // Austria
+  BE: true, // Belgium
+  BG: true, // Bulgaria
+  CH: true, // Switzerland
+  CZ: true, // Czech Republic
+  DE: true, // Germany
+  DK: true, // Denmark
+  ES: true, // Spain
+  EE: true, // Estonia
+  FI: true, // Finland
+  FR: true, // France
+  GB: true, // Great Britain / UK
+  GR: true, // Greece
+  HR: true, // Croatia
+  HU: true, // Hungary
+  IE: true, // Ireland
+  IS: true, // Iceland
+  IT: true, // Italy
+  LT: true, // Lithuania
+  LU: true, // Luxembourg
+  LV: true, // Latvia
+  NL: true, // Netherlands
+  NO: true, // Norway
+  PL: true, // Poland
+  PT: true, // Portugal
+  RO: true, // Romania
+  SE: true, // Sweden
+  SI: true, // Slovenia
+  SK: true  // Slovakia
+};
```

Comment:
Oh, nice.  Yeah, agreed on performance.

To keep the black boxness, I was thinking you would do 
```js
var isEULocale = countryCodes.indexOf(countryCode) !== -1;
```

Sorry if I wasn't clear on that.  I realize that means I wasn't actually comparing apples to apples :stuck_out_tongue_closed_eyes: